### DISCUSSION TITLE
'Show zones beginning with:' unavailable/greyed out

### DISCUSSION THREAD

**Discussion by IroHaturo** [poweradmin/poweradmin#433] [Q&A]
In list_zones.php I cannot click the letters, that usually would display zones starting w. a certain letter, see screenshots:

Initially [a] is active and [show all]
<img width="475" alt="Screenshot 2022-05-22 at 16 01 57" src="https://user-images.githubusercontent.com/57433686/169700162-42c5ca17-802c-46f6-9c22-e74b0cb1a2f6.png">

with [show all] clicked:
<img width="581" alt="Screenshot 2022-05-22 at 15 51 23" src="https://user-images.githubusercontent.com/57433686/169699215-38286b86-06c6-4c6d-b221-34c82608b0eb.png">

Same w. diff. browsers, so no cookie/cache issue

After gratefully using PDNS w. Poweradmin since many years, I just installed a phresh server from scratch, Debian 10 w. poweradmin v2.2.3, mysql  Ver 15.1 Distrib 10.3.34-MariaDB and PowerDNS Authoritative Server 4.1.6.
Things work fine, except the lil issue above.

Keep up the good work!
Cheers,
~iro

**Comment by RicardovanTongeren** [comment]
I've requested a pull request over here.

https://github.com/poweradmin/poweradmin/pull/436

This should fix the issue and after that can be closed.

**Comment by edmondas** [comment]
Thanks @RicardovanTongeren for the fix. Next time, I should be more careful about removing unclear code.

**Reply by edmondas** [reply]
@IroHaturo Check the latest release, and let me know if the issue persists.
